runbot_render_logs/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_runbot_render_logs_runbot_render_logs,runbot_render_logs.runbot_render_logs,model_runbot_render_logs_runbot_render_logs,,1,0,0,0
```

Comment:
I don't found the model `model_runbot_render_logs_runbot_render_logs` could you help me to see it?